components/views/src/main/res/layout/view_settings_left.xml

```diff
@@ -29,6 +21,9 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginStart="16dp"
+        android:ellipsize="end"
+        android:maxEms="12"
+        android:maxLines="1"
```

Comment:
Settings the `ems` value might be incorrect in some cases. I think, it would be great to move the content of this layout to where it included, and there we can use constraints to ellipsiphize text

Reply:
@tmedetbekov, Please take a look now.

Reply:
Looks very good. One more little thing, as we moved the content of this XML to `settings_item_switch` we can also duplicate this content to `view_settings_dropdown` and `view_settings_item` and totally remove this layout.

Reply:
@tmedetbekov This can not be done because the SettingsView.class is inflating it, so if we delete it we should delete the SettingsView.class

Reply:
I mean the `SettingsView` inflates `view_settings_item` layout which includes `view_settings_left`, so if we place the content of the `view_settings_left` to `view_settings_item` and `view_settings_dropdown` then we can remove it.

Reply:
@tmedetbekov  I think now it should be okay.

Reply:
Excellent @herou 

One tiny thing, can we remove the field `android:maxEms` now?

Reply:
@tmedetbekov  U have it.

Reply:
Nice work @herou! Thanks 👍